app/src/main/java/fr/free/nrw/commons/contributions/ContributionComparator.java

```diff
@@ -0,0 +1,11 @@
+package fr.free.nrw.commons.contributions;
+
+import java.util.Comparator;
+
+public class ContributionComparator implements Comparator<Contribution> {
+
+    @Override
+    public int compare(Contribution c1, Contribution c2) {
+        return c1.dateCreated.compareTo(c2.dateCreated);
```

Comment:
I think sorting using the upload date (`dateUploaded`) would be more appropriate here. It's my understanding that creation date is the date of creation of the media (image) while the upload date is the date of uploading the media. Correct me if I'm wrong.

Reply:
hello @sivaraam thanks for the feedback. I think you are right. I assumed the (dateCreated) was the date it was stored on the server. But in that case it doesn't differentiate from (dateUploaded). If you look at the contribution object returned, you realize  dateCreated and dateUploaded have the same value. I will change to dateUploaded for sorting just in case...

![contribution](https://user-images.githubusercontent.com/10153512/77824802-9530cd00-7105-11ea-806d-112a92ef9fe6.PNG)

Reply:
@sivaraam I reverted the changes I made. I think it's now clean to review... thanks

Reply:
> If you look at the contribution object returned, you realize dateCreated and dateUploaded have the same value.

You're right. For now both fields seem synonymous. Regardless, it's best to go with `dateUploaded` as I anticipate this changing with #2904.

Others might have a better knowledge about this.